src/clr-angular/data/datagrid/datagrid-filter.spec.ts

```diff
@@ -58,6 +58,15 @@ export default function(): void {
         filter.active = false;
         expect(component.active).toEqual(false);
       });
+
+      it('compares filters', function() {
+        const testFilter = new TestCustomStateFilter('test');
+        const anotherTestFilter = new TestCustomStateFilter('test');
+        const nonTestFilter = new TestCustomStateFilter('nonTest');
+        expect(testFilter.equals(anotherTestFilter)).toBeTruthy();
```

Comment:
I'm sorry, I don't understand this test. You are not testing any code from Clarity here, you're creating test objects and you're calling their method directly, that's not accomplishing anything.

Reply:
My initial idea was to play the scenario. But in the end even at that moment I had doubts...
You are right, it's totally pointless the way it is. Reverting.
Won't resolve the conversation in case someone can suggest a more meaningful test scenario for this case.

Reply:
Removed